aws-sdk-resources/lib/aws-sdk-resources/services/s3/object.rb

```diff
@@ -20,6 +20,10 @@ class Object
       #   performance improvements on large objects. Amazon S3 does
       #   not accept multipart copies for objects smaller than 5MB.
       #
+      # @options options [Integer] :content_length When included
+      #   the source object will not be queried for its size before
+      #   starting the copy.
+      #
```

Comment:
We should reword this documentation to indicate the `:content_length` is only needed when `:multipart_copy` is `true`.  The content_length is used for calculating the part size and nothing else.